src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -462,13 +462,6 @@ public static DetailAST parseWithComments(FileContents contents)
     }
 
     @Override
-    public void destroy() {
```

Comment:
@romani No other tests failed when chaining of destroy in TreeWalker to checks was removed.
We should atleast add a test to verify the chaining happens and isn't accidentally removed in the future.